modules/services/repositories/src/test/java/au/com/shiftyjelly/pocketcasts/models/type/SubscriptionMapper.kt

```diff
@@ -0,0 +1,21 @@
+package au.com.shiftyjelly.pocketcasts.models.type
+
+import com.android.billingclient.api.ProductDetails
+
+object SubscriptionMapper {
```

Comment:
Considering current code changes in this PR, it can be a bigger refactor to replace the existing `Subscription.Companion.fromProductDetails(...)` calling `SubscriptionMapper.map(...)` to use a DI object.

I can create a wrapper for `SubscriptionMapper` just for unit testing.  Do you think doing it just for testing will make it less complicated and error-prone? If not, I can refactor the code in a separate PR.

Reply:
Yes, using an interface or a regular class and mocking/faking will be much less error-prone and will not create confusion when navigating or modifying the code.